frontend/app/model/Eventbrite.scala

```diff
@@ -19,6 +19,15 @@ import scala.util.{Failure, Success, Try}
 
 object Eventbrite {
 
+  // This can be deleted once all these events have completed
+  val HiddenEvents = Map(
+    "18862189316" -> "19222944344",
+    "18882535171" -> "19223627387",
+    "18882579303" -> "19223798900",
+    "18882595351" -> "19223835008",
+    "18882606384" -> "19223927284"
```

Comment:
I realise that this is a quick and nasty fix to remove the bad events, but if this is likely to occur again in the future then we should add this redirect info into the old events meta data. After discussing with @rtyley this seems like something that would never happen again so i guess the quick fix is appropriate for now.